src/hooks/useWrapCallback.ts

```diff
@@ -37,14 +38,18 @@ export default function useWrapCallback(
     const sufficientBalance = inputAmount && balance && !balance.lessThan(inputAmount)
 
     if (inputCurrency === ETHER && currencyEquals(WETH[chainId], outputCurrency)) {
+      const { ether, weth } = getChainCurrencySymbols()
       return {
         wrapType: WrapType.WRAP,
         execute:
           sufficientBalance && inputAmount
             ? async () => {
                 try {
                   const txReceipt = await wethContract.deposit({ value: `0x${inputAmount.raw.toString(16)}` })
```

Comment:
can this function be moved outside somewhere? bit hard to read imo

Reply:
It can, but then I would either call the function always, not just when we deal with ETH//WETH token, or then I would need to complicate a bit the logic creating new consts like 

`const isEtherInput = inputCurrency === ETHER`


and then do sth like
```js
     if (isEthInput || isEthOutput)
      const { ether, weth } = getChainCurrencySymbols()
      // assign ether, weth to some let vars
      }
      
      // proceed with normal logic!

```

Reply:
I find it easier to read in my approach? what part specifically is hard to read?

Reply:
i just think it goes along with what we usually do, making constants and/or moving functions out when we dont need them. This function is also always being constructed on changes to memo which seems like a bit of a waste.

Also there are magic numbers with the `toSignificant`s no? why not use the currency decimals

Reply:
I think what u are saying is fair, although is not derived from this PR. It's the result of this code being repetitive. It could accept some refactoring. Let me work it out in a different PR

Reply:
I'm realising I modified the original Uniswap file! I'll try to override the hook instead. Although probably the refactor and this will probably come in a new PR